packages/fxa-auth-server/test/local/routes/totp.js

```diff
@@ -921,10 +1002,162 @@ describe('totp', () => {
   });
 
   // MFA-prefixed routes
+  describe('/mfa/totp/create', () => {
```

Comment:
I haven't been adding tests for these. If you look at what the testing code does, it just calls the handler directly. So you are essentially just double testing the handler function. If you wanted to check that that the route exists, I think that'd be fine, but duplicating the tests seems like overkill to me.